chat-openai-sample/README.md

```diff
@@ -56,3 +45,16 @@ Open the `.env` file to configure the following settings:
 4. (optional) To setup Event Grid, follow [Setup and host your Azure DevTunnel](#setup-and-host-your-azure-devtunnel) and register an Event Grid Webhook for the [ChatMessageReceived event](https://learn.microsoft.com/en-us/azure/event-grid/communication-services-chat-events#microsoftcommunicationchatmessagereceived-event) that points to your DevTunnel URI for `<DevTunnelUri>/api/chatMessageReceived`. See here for Event Grid Webhook configuration example [here](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/events/subscribe-to-events?pivots=platform-azp).
 
 Once that's completed you should have a running application. The best way to test this is to send a message in a chat thread to be analyzed by your intelligent agent.
+
+### Setup and host your Azure DevTunnel
```

Comment:
Can you put the optional tag on the heading instead?

Reply:
updated